pkg/cloudprovider/external_test.go

```diff
@@ -156,6 +156,13 @@ func TestIsCloudProviderExternal(t *testing.T) {
 			},
 		},
 		expected: true,
+	}, {
+		name: "FeatureSet: CustomNoUpgrade, Platform: IBMCloud",
+		status: &configv1.PlatformStatus{
+			Type: configv1.IBMCloudPlatformType,
+		},
+		featureGate: &configv1.FeatureGate{},
+		expected:    true,
```

Comment:
This test is named incorrectly
```suggestion
	}, {
		name: "No FeatureGate, Platform: IBMCloud",
		status: &configv1.PlatformStatus{
			Type: configv1.IBMCloudPlatformType,
		},
		featureGate: nil,
		expected:    true,
```

Reply:
Updated